pkg/injector/envoy_container.go

```diff
@@ -7,6 +7,7 @@ import (
 
```

Comment:
Why is this file being renamed, seems unnecessary as a part of this change.

Reply:
Moved the renames to this PR:  https://github.com/openservicemesh/osm/pull/1994  (So now #1991 is stacked on top of #1994)